frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/ThreeDotsMenu.tsx

```diff
@@ -4,10 +4,11 @@ import { MonitorIcon, TabsIcon } from '@navikt/aksel-icons';
 import { useTranslation } from 'react-i18next';
 import { repositoryPath } from 'app-shared/api/paths';
 import { GiteaIcon } from 'app-shared/icons';
-import { LegacyPopover, Button } from '@digdir/design-system-react';
+import { LegacyPopover } from '@digdir/design-system-react';
 import { MenuElipsisVerticalIcon } from '@navikt/aksel-icons';
 import { CloneModal } from './CloneModal';
-import { LocalChangesModal } from 'app-shared/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/LocalChangesModal';
+import { StudioButton } from '@studio/components';
+import { LocalChangesModal } from './LocalChangesModal/LocalChangesModal';
```

Comment:
```suggestion
import { LocalChangesModal } from './LocalChangesModal';
```

Reply:
This won't work as there's no index file in the folder. I can add it though, seems like there should be one there.